addons/website_event_track_quiz/static/src/js/event_quiz.js

```diff
@@ -242,17 +271,17 @@ var Quiz = publicWidget.Widget.extend({
      * @private
      */
     _onClickReset: function () {
+        let self = this;
         this._rpc({
             route: '/event_track/quiz/reset',
             params: {
                 event_id: this.track.eventId,
                 track_id: this.track.id
             }
-        }).then(function () {
-            window.location.reload();
+        }).then(function() {
+            self._resetQuiz();
```

Comment:
`.then(this._resetQuiz.bind(this))` would work I believe.

Reply:
Yes, it works :)